### PR TITLE
Feature setup mongo

### PR DESCRIPTION
**Pull request by alaindavis90**
Please review this.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+mongoose.connect('mongodb://localhost/profiledb', { useNewUrlParser: true })
+    .then(() => console.log('Connected to MongoDb...'))
+    .catch(err => console.error('Could not connect to MongoDB', err))
+
+const profileSchema = new mongoose.Schema({
+    name: String,
+    username: String,
+    email: String
+})
```
Comment (skassam21) [inline server/index.js]: This should be in a models folder

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/profiledb', { useNewUrlParser: true })
+    .then(() => console.log('Connected to MongoDb...'))
+    .catch(err => console.error('Could not connect to MongoDB', err))
+
+const profileSchema = new mongoose.Schema({
```
Comment (skassam21) [inline server/index.js]: I'd call this userSchema

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    name: String,
+    username: String,
+    email: String
+})
+
+const User = mongoose.model('Profiles', profileSchema);
```
Comment (skassam21) [inline server/index.js]: call this Users

Comment (skassam21) [reply server/index.js]: look up naming convention for collections in Mongodb (should it be plural or not)

File: server/routes/ping.js [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
-      response: `${teamName} is not part of the team. Modify your .env`
-    });
+  //if (teamName && process.env.TEAM_NAME.indexOf(teamName) >= 0) {
+  res.status(200).send({ response: "You are now registered" });
+  addUser(teamName);
+  //}
+  //else
+  // res.status(400).send({
+  //   response: `${teamName} is not part of the team. Modify your .env`
+  //// });
```
Comment (skassam21) [inline server/routes/ping.js]: avoid commented out code, delete it instead

File: server/db.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/profiledb', { useNewUrlParser: true })
+    .then(() => console.log('Connected to MongoDb...'))
+    .catch(err => console.error('Could not connect to MongoDB', err))
+
+const userSchema = new mongoose.Schema({
```
Comment (aeciorc) [inline server/db.js]: You need to set the attributes for these fields. For example, the email should be unique, both fields should be required

File: server/db.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    name: String,
+    email: String
+
+})
+
+const Users = mongoose.model('Users', userSchema);
+
```
Comment (aeciorc) [inline server/db.js]: You probably want to keep the logic below in a separate file, e.g inside a 'controllers' directory

File: server/db.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+const addUser = async (name, email) => {
```
Comment (aeciorc) [inline server/db.js]: this method should return something

File: server/db.js [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    const user = new Users({
+        name,
+        email
+    });
+    const result = await user.save();
+    console.log("user saved...", result);
+
+    getUsers();
```
Comment (aeciorc) [inline server/db.js]: remove this

File: server/db.js [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+const getUsers = async () => {
```
Comment (aeciorc) [inline server/db.js]: should also return something

File: server/routes/index.js [hunk 10]
```diff
@@ -1,8 +1,12 @@
-var express = require("express");
-var router = express.Router();
+let express = require("express");
+let router = express.Router();
+let addUser = require("../db")
 
-router.get("/welcome", function(req, res, next) {
-  res.status(200).send({ welcomeMessage: "Step 1 (completed)" });
+router.post("/", (req, res, next) => {
```
Comment (aeciorc) [inline server/routes/index.js]: this should be in a different route, and you need to validate this data before adding a user

File: server/routes/index.js [hunk 11]
```diff
@@ -1,8 +1,12 @@
...
+  res.status(200).send(req.body);
```
Comment (aeciorc) [inline server/routes/index.js]: Only return success if the user has successfully been created.

File: server/routes/ping.js [hunk 12]
```diff
@@ -1,15 +1,13 @@
...
-  else
-    res.status(400).send({
-      response: `${teamName} is not part of the team. Modify your .env`
-    });
+  res.status(200).send({ response: "You are now registered" });
+  //addUser(teamName);
+
 });
 
 module.exports = router;
```
Comment (aeciorc) [inline server/routes/ping.js]: Rename this file, it shouldn't be called ping anymore

**Pull request merged**